src/convertToObject.js

```diff
@@ -6,7 +6,18 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const stylesArray = sourceString.split(';');
+  const stylesObject = {};
```

Comment:
The variable name 'stylesObject' is too generic. A more descriptive name like 'styleRules' or 'cssProperties' would enhance readability and make the code's intent clearer.